addons/event/models/mail_template.py

```diff
@@ -9,17 +9,17 @@ class MailTemplate(models.Model):
     _inherit = 'mail.template'
 
     @api.model
-    def _name_search(self, name, domain=None, operator='ilike', limit=None, order=None):
+    def _search(self, domain, offset=0, limit=None, order=None, access_rights_uid=None):
```

Comment:
Isn't it better to do `*args, **kwargs` if we're not actually using most of these?

Reply:
seems better to me, and easier to maintain and know we are just touching domain

checking the code, of 22 overrides of `_search` in 17.0 we use the full signature 21 times, et 1 time `domain, *args, **kwargs`  but I don't see a reason we would not do it.